### PR TITLE
Migrate to Navigation 1.5.1.

### PR DESCRIPTION
**Pull request by yunikkk**
Previous navigation SDK version were inaccessible on Bintray, this PR fixes build

### CODE REVIEW

File: app/src/main/kotlin/com/mapbox/vision/teaser/ar/ArMapActivity.kt [hunk 1]
```diff
@@ -155,69 +145,65 @@ class ArMapActivity : AppCompatActivity(), MapboxMap.OnMapClickListener, OnMapRe
...
                     } else {
                         navigationMapRoute?.updateRouteVisibilityTo(false)
                     }
                     navigationMapRoute?.addRoute(currentRoute)
                 }
 
-                override fun onFailure(call: Call<DirectionsResponse>, throwable: Throwable) {}
-            })
+                override fun onRoutesRequestCanceled(routeOptions: RouteOptions) {
+                    TODO("not implemented")
```
Comment (kiryldz) [inline app/src/main/kotlin/com/mapbox/vision/teaser/ar/ArMapActivity.kt]: afair will cause a runtime crash if this code is hit, better just leave a comment here

File: app/src/main/kotlin/com/mapbox/vision/teaser/ar/ArMapActivity.kt [hunk 2]
```diff
@@ -155,69 +145,65 @@ class ArMapActivity : AppCompatActivity(), MapboxMap.OnMapClickListener, OnMapRe
...
+                }
+
+                override fun onRoutesRequestFailure(throwable: Throwable, routeOptions: RouteOptions) {
+                    TODO("not implemented")
```
Comment (kiryldz) [inline app/src/main/kotlin/com/mapbox/vision/teaser/ar/ArMapActivity.kt]: afair will cause a runtime crash if this code is hit, better just leave a comment here

**Review by kiryldz — APPROVED** [review]
If does work OK - let's merge it and iterate later if needed.
Not that familiar with API, generally LGTM

File: app/src/main/kotlin/com/mapbox/vision/teaser/ar/ArMapActivity.kt (hunk 1 above)
Comment (yunikkk) [reply app/src/main/kotlin/com/mapbox/vision/teaser/ar/ArMapActivity.kt]: Right

**Pull request merged**